### ISSUE TITLE
Deficient maven-clean-plugin configuration generated by wizard when maven layout is selected

### ISSUE THREAD

**Issue by corneliusludmann** [eclipse/xtext#1318]
Hi folks!

The pom.xml of the parent project generates a deficient pom.xml when maven source layout is used. The following maven-clean-plugin configuration is generated:

```
<plugin>
	<groupId>org.apache.maven.plugins</groupId>
	<artifactId>maven-clean-plugin</artifactId>
	<version>2.5</version>
	<configuration>
		<filesets>
			<fileset>
				<directory>${basedir}/src/main/xtend-gen</directory>
				<includes>
					<include>**/*</include>
				</includes>
				<directory>${basedir}/src/test/xtend-gen</directory>
				<includes>
					<include>**/*</include>
				</includes>
			</fileset>
		</filesets>
	</configuration>
</plugin>
```

However, mvn clean seems not work correctly because a new fileset needs to be openend after the first directory / includes block. The correct configuration should be like this:
```
<plugin>
	<groupId>org.apache.maven.plugins</groupId>
	<artifactId>maven-clean-plugin</artifactId>
	<version>2.5</version>
	<configuration>
		<filesets>
			<fileset>
				<directory>${basedir}/src/main/xtend-gen</directory>
				<includes>
					<include>**/*</include>
				</includes>
			</fileset>
			<fileset>
				<directory>${basedir}/src/test/xtend-gen</directory>
				<includes>
					<include>**/*</include>
				</includes>
			</fileset>
		</filesets>
	</configuration>
</plugin>
```

Not a big deal. But the folder src/main/xtend-gen will not be cleaned with the deficient configuration.

Bye,
Cornelius

**Comment by szarnekow** [comment]
@corneliusludmann Would you be willing to provide a small patch to fix the issue?

**Comment by cdietrich** [comment]
Fixed in 2.16
